book/HAS/hybrid-application-service-design.md

```diff
@@ -0,0 +1,88 @@
+# Hybrid Application Service (HAS) Design
+
+## Application CR Create
+
+HAC collects the information required for creating applications. An application is a collection of one or more components. Required information to be collected can be found in the API [doc](). When all the application information is available, HAC will create the Application CR to trigger the application creation. The HAS Application CR Operator will:
+1. Validate all the required fields in the Application CR
+2. If the app model repository and the GitOps repository repos are specified:
```

Comment:
A bit of a nit, but do we want to refer to the field names?  It did not immediately occur to me these were fields in the CR e.g. `if the Application.appModelRepository and Application.gitOpsRepository repos are specified:`